docs/android-startup-scenarios.md

```diff
@@ -0,0 +1,35 @@
+
+# Android Startup Test
+
+## Prereqs
+
+- Ensure `dotnet` is installed and available with the `dotnet` command for easy xharness installation. Any supported .NET Core version should work. [Dotnet Download](https://dotnet.microsoft.com/en-us/download) or [Daily Dotnet Download](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md)
+- Ensure `xharness` is installed and available with the `xharness` command. The current version in use can be found in the `eng/performance/maui_scenarios_android.proj` file at line 7 (under the tag `MicrosoftDotNetXHarnessCLIVersion`), although any recent version should work. [XHarness Install Instructions](https://github.com/dotnet/xharness?tab=readme-ov-file#installation-and-usage).
+- Have an Android app APK available for testing.
+- Have an Android Device (with developer mode enabled) or emulator connected to computer, and viewable with `xharness android device` or `xharness android adb -- devices -l`.
+
+## Steps
+
+1. Initialize the environment (note the . for bash):
+
+    ```sh
+    cd src/scenarios
+    . ./init.sh  # or `.\init.ps1` on Windows
```

Comment:
What about custom dotnet locations? 
Afaik there is/was `-dotnetdir` parameter which could specify custom dotnet installation.

Reply:
Added comment on this.